docs/network_disruption_hosts.md

```diff
@@ -1,17 +1,37 @@
-# Network disruption: Specifying hosts
+# Network disruption: Specifying hosts and services
 
 ## Q: When should I specify hosts?
 
 <p align="center"><kbd>
     <img src="../docs/img/network_hosts/notation_traffic.png" height=180 width=600 />
 </kbd></p>
 
-As with all disruptions, pods or nodes are targeted for injection if they satisfy the conditions of the label selector specified in the `selector` field. For network disruptions, we can also specify to only disrupt packets interacting with a particular host or set of hosts through the `network.hosts` field. We will refer to `network.hosts` field in the rest of the document as the `hosts` field.
+As with all disruptions, pods or nodes are targeted for injection if they satisfy the conditions of the label selector specified in the `selector` field. 
+For network disruptions, we can also specify to only disrupt packets interacting with a particular host or set of hosts through the `network.hosts` field. We will refer to `network.hosts` field in the rest of the document as the `hosts` field.
+The `hosts` field takes a list of `host`/`port`/`protocol` tuples. All three fields are optional.
 
 <p align="center"><kbd>
     <img src="../docs/img/network_hosts/notation_egress.png" height=160 width=570 />
 </kbd></p>
 
+## Q: When should I specify services instead?
+
+While the `network.hosts` field is meant for specifying only disrupting packets interacting with a particular IP, hostname, or CIDR range,
+a very common usecase of network disruptions is to disrupt packets interacting with a particular kubernetes service. This can be tricky for users,
```

Comment:
```suggestion
a very common use case of network disruptions is to disrupt packets interacting with a particular kubernetes service. This can be tricky for users,
```
I thought usecase was its own word but apparently it's not 😿